plugin/plugin.go

```diff
@@ -46,28 +46,17 @@ func (p *plugin) Validate(ctx context.Context, req *validator.Request) error {
 		return err
 	}
 
-	var message string
-	var values []string
-
 	for _, resource := range documents {
 		rs, err := r.Eval(ctx, rego.EvalInput(resource))
 		if err != nil {
 			return err
 		}
 		if rs[0].Bindings["deny"] == true {
-			s := strings.Split(fmt.Sprintf("%v", rs[0].Bindings["msg"]), ":")
-			if message == "" {
-				message = s[0]
-			}
-			values = append(values, s[1])
+			// TODO Return errors from multiple documents on a single message
+			message := fmt.Sprintf("pipeline %v", rs[0].Bindings["msg"])
+			return errors.New(message)
```

Comment:
I think this can be simplified to just return the policy error output:
```suggestion
			return errors.New(rs[0].Bindings["msg"])
```

Reply:
`errors.New()` only accepts strings and the result set bindings return interfaces, so we need some sort of `string()` conversion before returning the error.

Reply:
Oh gotcha, I was mostly pouting out that the code shouldn’t need to specify a “pipeline “ prefix

Reply:
Ah, you're right, I missed it! Will remove and keep the error message as it is.